neutron/assetlist.json

```diff
@@ -446,6 +446,32 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/babycorgi.png"
         }
       ]
-    }
+    },
+    {
+      "description": "Jimmy Neutron Finance",
+      "denom_units": [
+        {
+          "denom": "factory/neutron108x7vp9zv22d6wxrs9as8dshd3pd5vsga463yd/JIMMY",
+          "exponent": 0,
+          "aliases": [
+            "ujimmy"
+          ]
+        },
+        {
+          "denom": "jimmy",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/neutron108x7vp9zv22d6wxrs9as8dshd3pd5vsga463yd/JIMMY",
+      "name": "Jimmy",
+      "display": "Jimmy",
+      "symbol": "JIMMY",
+      "logo_URIs": {
+        "png": "https://jimmy-price.vercel.app/jimmy44.png"
```

Comment:
Please upload the image into this repo

Reply:
sorry, but its uploaded with the url what is needed furthermore?

Reply:
The logo image must be added to the neutron/images/ folder in this repo, and the URL must point to that upload--and not to an external website.

Reply:
sorry for bothering, but how can i add the image there? i tried creating folder and couldnt drop photo to save it

Reply:
also i cant upload file since it needs permission

Reply:
You have full permissions on your own fork. Be sure you're uploading to JIMMwe:patch-1 https://github.com/JIMMwe/chain-registry/tree/patch-1

Reply:
gotcha, done please check again